integration/framework/framework.go

```diff
@@ -95,21 +101,31 @@ const (
 	Unknown      string = ""
 )
 
+type RunArgs struct {
+	// Image to use.
+	Image string
+
+	// Arguments to the CLI.
+	Args []string
+
+	InnerArgs []string
```

Comment:
that's just a rename of the DockerRunArgs.  I assumed that args is to the runtime while innerargs are passed to the container?